src/components/DeprecatedBanner/DeprecatedBanner.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import styled from 'styled-components'
+import { ModalConsumer } from '../ModalManager/ModalManager'
+import { theme, Text, SafeLink, Button } from '@aragon/ui'
+import Banner from '../Banner/Banner'
+import Modal from '../Modal/Modal'
+
+const { negative, negativeText, accent, textDimmed } = theme
+
+const DeprecatedBanner = props => (
+  <ModalConsumer>
+    {({ showModal }) => <DeprecatedDao showModal={showModal} {...props} />}
+  </ModalConsumer>
+)
+
+class DeprecatedDao extends React.Component {
+  handleClick = () => {
+    const { dao, showModal } = this.props
+    showModal(DeprecatedModal, { dao })
+  }
+
+  render() {
+    const { children, dao } = this.props
+
+    return (
+      <React.Fragment>
+        <Banner
+          text="This Rinkeby DAO will be deprecated on 1/11/18"
+          textColor={negativeText}
+          onClick={this.handleClick}
+          buttonText="More info"
+          color={negative}
+          dao={dao}
+        />
+        {children}
+      </React.Fragment>
+    )
+  }
+}
+
+const DeprecatedBody = ({ dao }) => (
+  <React.Fragment>
+    <TopParagraph color={textDimmed}>
+      Over the last six months, we’ve made a lot of improvements to aragonOS.
+      Unfortunately a number of those improvements weren’t backwards-upgradeable
+      with the currently deployed contracts that you might have been using on
+      Rinkeby. So we’ve had to make the hard decision to deprecate those old
+      testnet DAOs. Please see{' '}
+      <StyledSafeLink
+        href="https://blog.aragon.one/0-5-dao-deprecation/"
+        target="_blank"
+      >
+        https://blog.aragon.one/0-5-dao-deprecation
+      </StyledSafeLink>{' '}
+      for more details.
```

Comment:
I wrote this copy a while back... so some edits for myself :smile:

- Unfortunately **,** (add a comma after "Unfortunately)
- with the ~~currently deployed~~ contracts ~~that you might have been using~~ **your DAOs were constructed with** on Rinkeby.
- ~~So we've had to make~~ **As a result, we've made** the ~~hard~~ **tough** decision to deprecate ~~those~~ **all** old testnet DAOs.